From 251de75543d2909ca9389800b41f6c493db5d988 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Wed, 3 Apr 2019 13:43:03 +0200
Subject: [PATCH] build: Fix meson error about subdir keyword

src/meson.build:83:0: ERROR: Subdir keyword must not be an absolute path.
---
 src/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- src/meson.build.orig	2018-07-24 13:29:49 UTC
+++ src/meson.build
@@ -86,7 +86,7 @@ headers = files(
 
 install_headers(
   headers,
-  subdir: join_paths(totem_includedir, totem_api_path)
+  subdir: totem_api_path
 )
 
 libtotem_player_sources = files(
